CHANGELOG.rst

```diff
@@ -912,21 +912,6 @@ in development
   easier since user simply needs to start sensor container with ``--debug`` flag and all the sensor
   logs with level debug or higher will be routed to the container log. (improvement)
 
-0.13.2 - September 09, 2015
```

Comment:
It was a duplicate (detected by the new lint target).

Probably from  a bad merge or similar.